Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/Utils/HttpHandlerHelper.cs

```
@@ -39,7 +41,14 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                 }
             }
 
-            httpResponse =  await base.SendAsync(request, cancellationToken);
+            try
+            {
+                httpResponse = await base.SendAsync(request, cancellationToken);
+            }
+            catch (Exception ex) {
+                this.ExceptionIntercepter?.Invoke(request, ex);
```

In-case of intetrceptor not configured, it results in exception dropped right? 

Is invoker expected to define the exception throw?

---

ohh yes..I should throw it as it is if intercepter is not configured.